core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -72,10 +77,32 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
     $scope.skillDifficultyLabel = getSkillDifficultyLabel();
     $scope.skillRubricExplanations = getRubricExplanation(
       $scope.skillDifficultyLabel);
+    $scope.reviewMessage = '';
+    $scope.suggestionIsRejected = suggestion.status === 'rejected';
+
+    const _getThreadHandlerUrl = function(suggestionId) {
+      return UrlInterpolationService.interpolateUrl(
+        '/threadhandler/<suggestionId>', { suggestionId });
+    };
+
+    const _getThreadMessagesAsync = function(threadId) {
+      return $http.get(_getThreadHandlerUrl(threadId)).then((response) => {
+        let threadMessageBackendDicts = response.data.messages;
+        return threadMessageBackendDicts.map(
+          m => ThreadMessage.createFromBackendDict(m));
+      });
+    };
 
     if (reviewable) {
       SiteAnalyticsService.registerContributorDashboardViewSuggestionForReview(
         'Question');
+    } else if ($scope.suggestionIsRejected) {
+      _getThreadMessagesAsync(suggestionId).then(
+        function(messageSummaries) {
+          $scope.reviewMessage = messageSummaries[1].text;
+          throw new Error('12312312');
```

Comment:
What is this error number?

Reply:
Oops this is unexpected. I was trying to test something. Reverted it! (and thanks for catching it!)